package.json

```diff
@@ -1,22 +1,23 @@
 {
+  "name": "apps-monorepo",
   "private": true,
+  "workspaces": [
+    "apps/*",
+    "packages/*"
+  ],
   "scripts": {
     "build": "turbo run build",
-    "dev": "turbo run dev",
-    "lint": "turbo run lint",
-    "format": "prettier --write \"**/*.{ts,tsx,md}\""
```

Comment:
also include this command for .js file extensions

Reply:
done